crates/astria-core/src/sequencer/v1alpha1/celestia.rs

```diff
@@ -32,7 +32,10 @@ impl CelestiaBlobBundle {
             rollup_ids_proof,
         } = block.into_unchecked();
 
-        let tree = super::derive_merkle_tree_from_rollup_txs(&rollup_transactions);
+        let rollup_base_transactions = rollup_transactions
```

Comment:
Is all of this still necessary if the individual`RollupTransaction` types also contain proofs?

Reply:
It's a different proof we are constructing here, and is needed for Celestia metadata blob posting I believe.

Reply:
I believe it should be the same proof but currently isn't. I have re-read and added a comment to the loop in `try_from_cometbft` that constructs the individual proofs. I believe as it stands the construction is not correct.

Reply:
fixed this, should be correct now, adding unit tests also